Makefile

```diff
@@ -48,6 +49,12 @@ bin/atc: src/log/*.go src/atc/*.go
 	@mkdir -p bin
 	$(BUILD) -o $@ ${SRC}/atc
 
+jsx: src/react/jsx/*.js
+	@cd src/react && $(NPM) run build-js
```

Comment:
logging was available though, only the line showing what would be executed was missing. e.g `cd src/react && npm run build-js`